.github/workflows/build-complete-samples.yml

```diff
@@ -58,7 +58,7 @@ jobs:
 
             - project_path: 'samples/bot-calling-meeting/csharp/CallingBotSample.csproj'
               name: 'bot-calling-meeting'
-              version: '3.1.301'
```

Comment:
@eoinobrien  - Updated the framework version in build. 

https://github.com/OfficeDev/Microsoft-Teams-Samples/actions/runs/3485879473